tracing/README.md

```diff
@@ -57,6 +57,8 @@ First, add this to your `Cargo.toml`:
 tracing = "0.1.10"
 ```
 
+*Compiler support: requires rustc 1.39+*
```

Comment:
We should probably add this to the docs for _all_ crates in the project.

A part of me wants to have separate MSRV for each crate, since they don't _all_ have the same requirements, but managing that on CI would be incredibly painful, and I'd rather have a MSRV that we can actually test support for...

Reply:
I'll add it everywhere that has ```toml at a minimum because those are easy to find.

Reply:
or [dependencies]